Python-Resources.md

```diff
@@ -37,3 +47,26 @@
 * [/r/Python](https://www.reddit.com/r/python)
 * [/r/LearnPython](https://www.reddit.com/r/learnpython)
 * [Planet Python](http://planetpython.org/)
+
+## Blogs
+
+* [Real Python Blog](https://realpython.com/blog/)
+* [Python Tips](http://pythontips.com/)
+* [Schooling The N00b](https://schoolingthenoob.wordpress.com/)
+
+## Podcasts
+
+* [Podcast.__init__](http://pythonpodcast.com/)
```

Comment:
The markdown rendered it just into a bold Podcast.__init__, better write it as `Podcast.\__init__` -> Podcast.\__init__